### PR TITLE
Add `suggestedOptionComponent` property, update README, Fix #44

### LINKED ISSUE #44: Feature request: yield option.__isSuggestion__
Currently if the `option.__isSuggestion__`, ember-power-select-with-create takes over. 

For my use case, I want to show an icon and some other helpful information to the user if they are seeing the create option.

I'm requesting that there be some kind boolean property that I can set, (default `false`), that when `true` would yield the suggested option just like any other option.

PR to follow.

### PR DESCRIPTION
**Pull request by zkwentz**
Adds the ability to specify a `suggestedOptionComponent` that will receive two properties, `option` (the suggested option) and `term` (the current search term).

### CODE REVIEW

**Comment by k-fish** [comment]
This would be great if it could be included, I was actually surprised to see that the suggestion could only be text (given that everything else in EPS is contextual components).

**Comment by cibernox** [comment]
There is only one more change and I could merge this.

File: addon/components/power-select-with-create/suggested-option.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+import layout from '../../templates/components/power-select-with-create/suggested-option';
+
+export default Ember.Component.extend({
+  layout
```
Comment (zkwentz) [inline addon/components/power-select-with-create/suggested-option.js]: My bad, didn't see this, k.

Update coming.

**Comment by zkwentz** [comment]
@cibernox I didn't see this update. Sorry for the delay, modified.

**Comment by cibernox** [comment]
Released in 0.3.0

**Pull request merged**